make qt-lib missing

<!--
Hello and thanks for contributing! To help us diagnose your problem quickly, please:

 - Include a minimal demonstration of the bug, including code, logs, and screenshots.
 - Ensure you can reproduce the bug using the latest release.
 - Only post to report a bug or request a feature; direct all other questions to: https://stackoverflow.com/questions/tagged/mapbox
-->

**Platform:** OSX
**Mapbox SDK version:** Current master / `3.3.3` in `package.json`

### Steps to trigger behavior

 1. Checkout repository
 2. Run `make qt-lib`

### Expected behavior

Building the library.

### Actual behavior

An error:

```sh
make: *** No rule to make target `qt-lib'.  Stop.
```

--- a/Makefile
+++ b/Makefile
@@ -387,6 +387,10 @@ qtproj: $(MACOS_QT_PROJ_PATH)
 
 endif
 
+.PHONY: qt-lib
+qt-lib: $(QT_BUILD)
+	$(NINJA) $(NINJA_ARGS) -j$(JOBS) -C $(QT_OUTPUT_PATH) qmapboxgl
+
 .PHONY: qt-app
 qt-app: $(QT_BUILD)
 	$(NINJA) $(NINJA_ARGS) -j$(JOBS) -C $(QT_OUTPUT_PATH) mbgl-qt

